pkg/queue/queue_test.go

```
@@ -26,7 +28,7 @@ import (
 )
 
 func TestFIFOQueue(t *testing.T) {
-	q := newQueue()
+	q, _ := newQueue()
```

don't silence the error